src/main/java/org/kohsuke/github/Requester.java

```diff
@@ -241,8 +242,29 @@ public void to(String tailApiUrl) throws IOException {
         return method(method).to(tailApiUrl, type);
     }
 
-    @SuppressFBWarnings("SBSC_USE_STRINGBUFFER_CONCATENATION")
     private <T> T _to(String tailApiUrl, Class<T> type, T instance) throws IOException {
+        int retries = 5;
+        while (true) {
+            try {
+                return __to(tailApiUrl, type, instance);
+            } catch (IOException e) {
```

Comment:
No, because `HttpException` extends from `IOException`.

I will limit the retry mechanism to only `HttpException`s with status codes >= 400 && `SocketException`-- I feel that there's merit to retrying other socket exceptions, such as connection reset errors.